quelpa.el

```diff
@@ -156,6 +162,9 @@ If nil the update is disabled and the repo is only updated on
 (defvar quelpa-recipe '(quelpa :url "https://framagit.org/steckerhalter/quelpa.git" :fetcher git)
   "The recipe for quelpa.")
 
+(defvar quelpa--use-current-state-p nil
```

Comment:
As noted in another comment, this variable and `quelpa-force-upgrade-p` seem to overlap or conflict.

Reply:
Please refer to my other comment for detail explain for this